Taxi/Taxi/Presenter/CalendarModal.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  PostView.swift
+//  Planterior_App
+//
+//  Created by  Isac Park on 2022/05/03.
+//
+
+import SwiftUI
+
+struct CalendarModal: View {
+    @Binding var isShowing: Bool
+    @State private var isPresented = false
+    @State private var curHeight: CGFloat = 400
+    private let minHeight: CGFloat = 300
+    private let maxHeight: CGFloat = 300
+    private let startOpacity: Double = 0.4
+    private let endOpacity: Double = 0.8
+
+    var body: some View {
+        ZStack(alignment: .bottom) {
+            if isShowing {
+                Color.black
+                    .opacity(0.3)
+                    .ignoresSafeArea()
+                    .onTapGesture {
+                        withAnimation(.easeInOut) {
+                            isShowing = false
+                        }
+                    }
+                mainView
+                    .transition(.move(edge: .bottom))
+            }
+        }
+        .frame(maxWidth: .infinity, maxHeight: .infinity, alignment: .bottom)
+        .ignoresSafeArea()
+    }
+
+    var mainView: some View {
+        VStack {
+            ZStack {
+                Capsule()
+                    .frame(width: 40, height: 6)
+            }
+            .frame(height: 40)
+            .frame(maxWidth: .infinity)
+            .background(Color.white.opacity(0.00001))
+            NavigationView {
+            } // 차후 달력 컴포넌트와 연동 예정
```

Comment:
```suggestion
            } // TODO: 차후 달력 컴포넌트와 연동 예정
```

TODO로 되어있으면 다른 팀원도 어느 타이밍에 달력 컴포넌트가 추가되는지 알아보기 쉬울 것 같아용

Reply:
오, 생각지 못했는데 알려주셔서 감사합니다. 반영 완료하였습니다~!!